src/Uno.UI/UI/Xaml/Shapes/Polyline.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Windows.UI.Xaml.Media;
+
+
+namespace Windows.UI.Xaml.Shapes
+{
+	public partial class Polyline : ArbitraryShapeBase
+	{
+		#region Points
+		public PointCollection Points
+		{
+			get { return (PointCollection)GetValue(PointsProperty); }
+			set { SetValue(PointsProperty, value); }
+		}
+
+		public static global::Windows.UI.Xaml.DependencyProperty PointsProperty { get; } =
+			DependencyProperty.Register(
+				"Points", typeof(global::Windows.UI.Xaml.Media.PointCollection),
+				typeof(global::Windows.UI.Xaml.Shapes.Polyline),
+				new FrameworkPropertyMetadata(
+					defaultValue: default(global::Windows.UI.Xaml.Media.PointCollection),
```

Comment:
Is it null by defaut on Windows ?

Reply:
I checked on UWP and Point is null by Default when we created a shape in code behind for example.